pallets/phala/src/lib.rs

```diff
@@ -16,6 +16,8 @@ use frame_support::{
 };
 use codec::Decode;
 
+//use xcm_adapter::{XcmExecutor};
+
```

Comment:
Are we waiting for the real XcmExecutor from Rococo / Polkadot? If not, we can remove this line